Hoxtygen/next-fullstack
# Feat: show user listings total income and rented listings (HOM-141)

## What does this PR do?
* Show users' listing statistics

## Description of tasks to be completed
* Create components  for show stat values
* Create chart for data breakdown

## How should this be manually tested?
* Clone the repo
* Open a terminal on your PC and run `git clone <repo-url>`
* `cd` into the project directory and run `yarn install`
* Run `yarn dev` to start the server
* Open a browser and navigate to `http://localhost:3000/`
* Sign up as a new user or log in if you already have one.
* You should see the stats on the dashboard.


##What are the relevant Jira board stories
[HOM-141](https://wasiu-dev.atlassian.net/browse/HOM-141)

[HOM-141]: https://wasiu-dev.atlassian.net/browse/HOM-141?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
